bin/compile

```diff
@@ -15,6 +15,9 @@ build_path=$(cd $1 && pwd)
 cache_path=$(cd $2 && pwd)
 env_path=$(cd $3 && pwd)
 
+export BUILDPACK_BUILD_PATH=${build_path}
+export BUILDPACK_CACHE_PATH=${cache_path}
+export BUILDPACK_ENV_PATH=${env_path}
```

Comment:
We don't need to export `env_path`, all env vars in there are exported by the buildpack.